src/wallet/walletdb.cpp

```diff
@@ -23,6 +23,44 @@
 
 #include <boost/thread.hpp>
 
+namespace DBKeys {
+    const std::string BESTBLOCK_NOMERKLE{"bestblock_nomerkle"};
+    const std::string BESTBLOCK{"bestblock"};
+    const std::string CRYPTED_KEY{"ckey"};
+    const std::string CSCRIPT{"cscript"};
+    const std::string DEFAULTKEY{"defaultkey"};
+    const std::string DESTDATA{"destdata"};
+    const std::string HDCHAIN{"hdchain"};
+    const std::string KEYMETA{"keymeta"};
+    const std::string KEY{"key"};
+    const std::string MASTER_KEY{"mkey"};
+    const std::string MINVERSION{"minversion"};
+    const std::string NAME{"name"};
+    const std::string OLD_KEY{"wkey"};
+    const std::string ORDERPOSNEXT{"orderposnext"};
+    const std::string POOL{"pool"};
+    const std::string PURPOSE{"purpose"};
+    const std::string TX{"tx"};
+    const std::string VERSION{"version"};
+    const std::string WATCHS{"watchs"};
+
+    // Sapling
+    const std::string SAP_KEYMETADA{"sapzkeymeta"};
+    const std::string SAPKEY{"sapzkey"};
+    const std::string CRYPTED_SAPKEY{"csapzkey"};
```

Comment:
nit: could be good to use the same prefix (`SAP_`) for all related db keys
```suggestion
    const std::string SAP_KEYMETA{"sapzkeymeta"};
    const std::string SAP_KEY{"sapzkey"};
    const std::string SAP_KEY_CRIPTED{"csapzkey"};
```